tasks/install_packages.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Install required packages
+  apt:
+    name:
+      - php-mysql
+      - python-mysqldb
+      - php-ldap
+      - php-imagick
+      - php-gd
+      - php-mbstring
+      - php-mcrypt
+      - php-curl
+  when: ansible_pkg_mgr == "apt"
```

Comment:
Why this check? The role is only supported on apt systems

Reply:
Not necessary in the current state but makes it a little bit easier to add alternative installation tasks for other package managers